chat-plugins/hangman.js

```diff
@@ -135,19 +135,8 @@ class Hangman extends Rooms.RoomGame {
 		output += '<p style="text-align:left;font-weight:bold;font-size:10pt;margin:5px 0 0 15px">' + (result === 1 ? 'Too bad! The mon has been hanged.' : (result === 2 ? 'The word has been guessed. Congratulations!' : 'Hangman')) + '</p>';
 		output += '<table><tr><td style="text-align:center;">' + this.hangingMan() + '</td><td style="text-align:center;width:100%;word-wrap:break-word">';
 
-		let wordString = '';
-		if (result === 1) {
-			for (let i = 0; i < this.wordSoFar.length; i++) {
-				if (i) wordString += '<small>&nbsp;</small>';
-				if (this.wordSoFar[i] === '_') {
-					wordString += '<font color="#7af87a">' + this.word[i] + '</font>';
-				} else {
-					wordString += this.wordSoFar[i];
-				}
-			}
-		} else {
-			wordString = this.wordSoFar.join('<small>&nbsp;</small>');
-		}
+		let wordString = this.wordSoFar.join('&nbsp;');
+		if (result === 1) wordString = wordString.replace(/_/g, '<font color="#7af87a">_</font>');
```

Comment:
This doesn't do the same thing at all; the point is that we're supposed to replace `_` with the actual letter that belongs there if the result is `1` (i.e. if the game ends, possibly in a failure)